pills/10/hello-nix.txt

```diff
@@ -2,6 +2,6 @@ let
   pkgs = import <nixpkgs> {};
   mkDerivation = import ./autotools.nix pkgs;
 in mkDerivation {
-  name = "hello";
-  src = ./hello-2.10.tar.gz;
-}
+    name = "hello";
+    src = ./hello-2.12.1.tar.gz;
```

Comment:
I suppose indentation was fine, unless some took did that dir you. Me being not up to date with conventions...